jellyfin_kodi/helper/api.py

```diff
@@ -32,20 +32,14 @@ def get_playcount(self, played, playcount):
 
     def get_naming(self):
 
-        if self.item['Type'] == 'Episode':
+        if self.item['Type'] == 'Episode'and 'SeriesName' in self.item:
```

Comment:
Missing space around `and`

Reply:
Damnit. Only looked through the code changes 3 times :/